docs/book/src/developer/providers/implementers-guide/building_running_and_testing.md

```diff
@@ -28,16 +28,17 @@ kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/
 
 ### Cluster API
 
-Before you can deploy the infrastructure controller, you'll need to deploy Cluster API itself.
+Before you can deploy the infrastructure controller, you'll need to deploy Cluster API itself to the management cluster.
 
-You can use a precompiled manifest from the release page, or clone [`cluster-api`][capi] and apply its manifests using `kustomize`.
+You can use a precompiled manifest from the [release page](releases), run `clusterctl init`, or clone [`cluster-api`][capi] and apply its manifests using `kustomize`:
```

Comment:
```suggestion
You can use a precompiled manifest from the [release page][releases], run `clusterctl init`, or clone [`cluster-api`][capi] and apply its manifests using `kustomize`:
```
otherwise it points to the releases file and thus gives a 404

Reply:
done